### PR TITLE
12.0 Migration School Module

### PR DESCRIPTION
**Pull request by AnuKPatel**
Hello,

Migration of school module is done.
Please review and merge the code.

### CODE REVIEW

File: school/wizard/teriminate_reason.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 class TerminateReason(models.TransientModel):
     _name = "terminate.reason"
+    _description = " Terminate Reason"
```
Comment (dhirendrasinh-Serpent) [inline school/wizard/teriminate_reason.py]: Fixe pylint

**Review by dhirendrasinh-Serpent — COMMENTED** [review]
Improved code

File: school/views/school_view.xml [hunk 2]
```diff
@@ -332,10 +332,10 @@
                 </group>
                 <newline />
                 <group expand="0" string="Group By..." colspan="12" col="10">
-                    <filter icon="terp-project" string="School" help="By School" context="{'group_by':'school_id'}" />
-                    <filter icon="terp-project" string="Class" help="By Class" context="{'group_by':'standard_id'}" />
-                    <filter icon="terp-project" string="Division" help="By Division" context="{'group_by':'division_id'}" />
-                    <filter icon="terp-project" string="Medium" help="By Medium" context="{'group_by':'medium_id'}" />
+                    <filter name="school" icon="terp-project" string="School" help="By School" context="{'group_by':'school_id'}" />
```
Comment (dhirendrasinh-Serpent) [inline school/views/school_view.xml]: No need icon attribute in filter V12

File: school/views/school_view.xml [hunk 3]
```diff
@@ -332,10 +332,10 @@
...
+                    <filter name="class" icon="terp-project" string="Class" help="By Class" context="{'group_by':'standard_id'}" />
```
Comment (dhirendrasinh-Serpent) [inline school/views/school_view.xml]: No need icon attribute in filter V12

File: school/views/school_view.xml [hunk 4]
```diff
@@ -332,10 +332,10 @@
...
+                    <filter name="division" icon="terp-project" string="Division" help="By Division" context="{'group_by':'division_id'}" />
```
Comment (dhirendrasinh-Serpent) [inline school/views/school_view.xml]: No need icon attribute in filter V12

File: school/views/school_view.xml [hunk 5]
```diff
@@ -332,10 +332,10 @@
...
+                    <filter name="medium" icon="terp-project" string="Medium" help="By Medium" context="{'group_by':'medium_id'}" />
```
Comment (dhirendrasinh-Serpent) [inline school/views/school_view.xml]: No need icon attribute in filter V12

File: school/views/student_view.xml [hunk 6]
```diff
@@ -300,16 +300,16 @@
                 </group>
                 <newline />
                 <group expand="0" string="Group By..." colspan="12" col="10">
-                    <filter icon="terp-project" string="Class" help="By Class" context="{'group_by':'standard_id'}" />
-                    <filter icon="terp-project" string="Gender" help="By Gender" context="{'group_by':'gender'}" />
-                    <filter icon="terp-project" string="Name" help="By Class" context="{'group_by':'student_name'}" />
-                    <filter icon="terp-project" string="School" help="By School" context="{'group_by':'school_id'}" />
-                    <filter icon="terp-project" string="State" help="By State" context="{'group_by':'state'}"/>
+                    <filter name="class" icon="terp-project" string="Class" help="By Class" context="{'group_by':'standard_id'}" />
```
Comment (dhirendrasinh-Serpent) [inline school/views/student_view.xml]: No need icon attribute in filter V12

File: school/views/student_view.xml [hunk 7]
```diff
@@ -300,16 +300,16 @@
...
+                    <filter name="gender" icon="terp-project" string="Gender" help="By Gender" context="{'group_by':'gender'}" />
```
Comment (dhirendrasinh-Serpent) [inline school/views/student_view.xml]: No need icon attribute in filter V12

File: school/views/student_view.xml [hunk 8]
```diff
@@ -300,16 +300,16 @@
...
+                    <filter name="name" icon="terp-project" string="Name" help="By Class" context="{'group_by':'student_name'}" />
```
Comment (dhirendrasinh-Serpent) [inline school/views/student_view.xml]: No need icon attribute in filter V12

File: school/views/student_view.xml [hunk 9]
```diff
@@ -300,16 +300,16 @@
...
+                    <filter name="school" icon="terp-project" string="School" help="By School" context="{'group_by':'school_id'}" />
```
Comment (dhirendrasinh-Serpent) [inline school/views/student_view.xml]: No need icon attribute in filter V12

File: school/views/student_view.xml [hunk 10]
```diff
@@ -300,16 +300,16 @@
...
+                    <filter name="state" icon="terp-project" string="State" help="By State" context="{'group_by':'state'}"/>
```
Comment (dhirendrasinh-Serpent) [inline school/views/student_view.xml]: No need icon attribute in filter V12

File: school/views/student_view.xml [hunk 11]
```diff
@@ -300,16 +300,16 @@
...
                 </group>
                 <separator/>
-                <filter icon="terp-project" string="Alumni" domain="[('state','=', 'alumni')]" help="Admission In Confirm State"/>
-                <filter icon="terp-project" string="Terminated" domain="[('state','=', 'terminate')]" help="Admission In Pending State"/>
-                <filter icon="terp-project" string="Done" domain="[('state','=', 'done')]" help="Admission In Done State"/>
+                <filter name="alumni" icon="terp-project" string="Alumni" domain="[('state','=', 'alumni')]" help="Admission In Confirm State"/>
```
Comment (dhirendrasinh-Serpent) [inline school/views/student_view.xml]: No need icon attribute in filter V12

File: school/views/student_view.xml [hunk 12]
```diff
@@ -300,16 +300,16 @@
...
+                <filter name="terminate" icon="terp-project" string="Terminated" domain="[('state','=', 'terminate')]" help="Admission In Pending State"/>
```
Comment (dhirendrasinh-Serpent) [inline school/views/student_view.xml]: No need icon attribute in filter V12

File: school/views/student_view.xml [hunk 13]
```diff
@@ -300,16 +300,16 @@
...
+                <filter name="done" icon="terp-project" string="Done" domain="[('state','=', 'done')]" help="Admission In Done State"/>
```
Comment (dhirendrasinh-Serpent) [inline school/views/student_view.xml]: No need icon attribute in filter V12

File: school/views/student_view.xml [hunk 14]
```diff
@@ -620,10 +620,10 @@
                 <field name="school_id" />
                 <newline />
                 <group expand="0" string="Group By..." colspan="12" col="10">
-                    <filter icon="terp-project" string="Class" domain="[]" help="By Class" context="{'group_by':'standard_id'}" />
-                    <filter icon="terp-project" string="Gender" domain="[]" help="By Gender" context="{'group_by':'gender'}" />
-                    <filter icon="terp-project" string="Name" domain="[]" help="By Class" context="{'group_by':'student_name'}" />
-                    <filter icon="terp-project" string="School" help="By School" context="{'group_by':'school_id'}" />
+                    <filter name="class" icon="terp-project" string="Class" domain="[]" help="By Class" context="{'group_by':'standard_id'}" />
```
Comment (dhirendrasinh-Serpent) [inline school/views/student_view.xml]: No need icon attribute in filter V12

File: school/views/student_view.xml [hunk 15]
```diff
@@ -620,10 +620,10 @@
...
+                    <filter name="gender" icon="terp-project" string="Gender" domain="[]" help="By Gender" context="{'group_by':'gender'}" />
```
Comment (dhirendrasinh-Serpent) [inline school/views/student_view.xml]: No need icon attribute in filter V12

File: school/views/student_view.xml [hunk 16]
```diff
@@ -620,10 +620,10 @@
...
+                    <filter name="name" icon="terp-project" string="Name" domain="[]" help="By Class" context="{'group_by':'student_name'}" />
```
Comment (dhirendrasinh-Serpent) [inline school/views/student_view.xml]: No need icon attribute in filter V12

File: school/views/student_view.xml [hunk 17]
```diff
@@ -620,10 +620,10 @@
...
+                    <filter name="school" icon="terp-project" string="School" help="By School" context="{'group_by':'school_id'}" />
```
Comment (dhirendrasinh-Serpent) [inline school/views/student_view.xml]: No need icon attribute in filter V12

**Review by dhirendrasinh-Serpent — COMMENTED** [review]
Improved code

File: school/views/school_view.xml (hunk 2 above)
Comment (dhirendrasinh-Serpent) [inline school/views/school_view.xml]: No need icon attribute in filter V12

File: school/views/school_view.xml (hunk 3 above)
Comment (dhirendrasinh-Serpent) [inline school/views/school_view.xml]: No need icon attribute in filter V12

File: school/views/school_view.xml (hunk 4 above)
Comment (dhirendrasinh-Serpent) [inline school/views/school_view.xml]: No need icon attribute in filter V12

File: school/views/school_view.xml (hunk 5 above)
Comment (dhirendrasinh-Serpent) [inline school/views/school_view.xml]: No need icon attribute in filter V12

File: school/views/student_view.xml (hunk 6 above)
Comment (dhirendrasinh-Serpent) [inline school/views/student_view.xml]: No need icon attribute in filter V12

File: school/views/student_view.xml (hunk 7 above)
Comment (dhirendrasinh-Serpent) [inline school/views/student_view.xml]: No need icon attribute in filter V12

File: school/views/student_view.xml (hunk 8 above)
Comment (dhirendrasinh-Serpent) [inline school/views/student_view.xml]: No need icon attribute in filter V12

File: school/views/student_view.xml (hunk 9 above)
Comment (dhirendrasinh-Serpent) [inline school/views/student_view.xml]: No need icon attribute in filter V12

File: school/views/student_view.xml (hunk 10 above)
Comment (dhirendrasinh-Serpent) [inline school/views/student_view.xml]: No need icon attribute in filter V12

File: school/views/student_view.xml (hunk 11 above)
Comment (dhirendrasinh-Serpent) [inline school/views/student_view.xml]: No need icon attribute in filter V12

File: school/views/student_view.xml (hunk 12 above)
Comment (dhirendrasinh-Serpent) [inline school/views/student_view.xml]: No need icon attribute in filter V12

File: school/views/student_view.xml (hunk 13 above)
Comment (dhirendrasinh-Serpent) [inline school/views/student_view.xml]: No need icon attribute in filter V12

File: school/views/student_view.xml (hunk 14 above)
Comment (dhirendrasinh-Serpent) [inline school/views/student_view.xml]: No need icon attribute in filter V12

File: school/views/student_view.xml (hunk 15 above)
Comment (dhirendrasinh-Serpent) [inline school/views/student_view.xml]: No need icon attribute in filter V12

File: school/views/student_view.xml (hunk 16 above)
Comment (dhirendrasinh-Serpent) [inline school/views/student_view.xml]: No need icon attribute in filter V12

File: school/views/student_view.xml (hunk 17 above)
Comment (dhirendrasinh-Serpent) [inline school/views/student_view.xml]: No need icon attribute in filter V12

**Review by dhirendrasinh-Serpent — COMMENTED** [review]
Improved code

**Comment by AnuKPatel** [comment]
Removed icon attribute from Filter tag.

File: school/wizard/teriminate_reason.py (hunk 1 above)
Comment (AnuKPatel) [reply school/wizard/teriminate_reason.py]: Fixed

**Pull request merged**